readme.md

```diff
@@ -373,7 +373,48 @@ entities:
     extend_to_present: true # false by default for statistics
 ```
 
-### `lambda:` transforms
+### `filters:`
```

Comment:
@FrnchFrgg , @zanna-37 I'd really appreciate your opinion about this feature I'm working on. Particularly on the API, any filters you think would be cool to add and (if you have the time and interest) the implementation.

Looking forward to your feedback :)

Reply:
I'm excited you are bringing this feature to the card. 👍🏻
I gave a quick look and didn't find obvious issues. I'll try to test the new implementation but no promises because this period of the year is dense with things to do. 🙈

Reply:
Thanks!
Oh, I wasn't expecting testing, mostly comments on the api & such. :)

The pr is definitely not final. One issue I didn't quite solve is the fact that the data comes in as strings, including"unavailable", and it is usually more convenient to handle them as numbers without nulls or gaps. The direction I took in this PR is to start by parsing everything as floats and filtering out all non numbers and gaps, but this makes filters useless for non numbers (like a door state and whatnot). And it also hides the gaps where entities become unavailable

Reply:
Mmm, that not good news =/

Reply:
I meant only when using filters, but yeah, it's not great. Another option is for the filters to take non floats into account and leave them there, and have multiple versions of the map filters (e.g. `map_y_numeric`, and `map_y_raw`), it just feels kind of complex to grasp for the user

Reply:
Ok, this has scope creeped a lot, but I think I managed to provide both the flexibility of dealing with strings and missing values if the user wants to, while also adding extra filters to handle numbers only without loosing the gaps in the plot from missing values.

I'm starting to wrap it up. I'm even planning to deprecate lambdas (I know there are a lot of examples using lambdas, but I think it will be worth it)
I'd highly appreciate your opinion about this section of the readme:
https://github.com/dbuezas/lovelace-plotly-graph-card/blob/feat/add-esphome-style-filters/readme.md#filters

Reply:
btw, one of the params is "attributes" which is only used to pass  unit_of_measurement in and out. I'll change that because it looks like the entity attributes of each datapoint, which is inside state.attributes.

Reply:
Looks great!

On the readme, you have map_y_numeric on line #400, but then in the examples from line #433 onward it refers to map_y_numbers - looks like a typo in one or the other??

Reply:
Good catch! Ill fix that. I bet there are more of those. I haven't started testing this and added a lot of stuff without even running it so the final stretch will be bumpy for me :)

Reply:
Btw, which name sounds more intuitive to you?

Reply:
I think map_y_numeric, but I wouldn't put a lot of effort into changing it if you have already implemented map_y_numbers!

Reply:
pre-release available!
https://github.com/dbuezas/lovelace-plotly-graph-card/releases/tag/v2.0.0-beta